docs/framework/guides/document-editor.md

```diff
@@ -58,9 +61,13 @@ The code you have just created will run the editor but still, the user interface
 The following structure has two containers that correspond to the configuration you have just used. The editor will inject the toolbar and editable into respective containers as it starts.
 
 ```html
-<div class="document-editor">
+<div class="document-editor ck-rounded-corners">
```

Comment:
Why is it required to add this class manually? This is unclear for me why, as an integration developer, I'd need to add some CKE's class to my elements to make buttons in the toolbar rounded.

Reply:
Fixed in https://github.com/ckeditor/ckeditor5-editor-decoupled/pull/11/commits/63683b6e1ba041607fb830b0c7896a5a0c88a07c.